accord-core/src/main/java/accord/local/Node.java

```diff
@@ -221,12 +221,10 @@ public Node(Id id, MessageSink messageSink,
         this.random = random;
         this.persistDurableBefore = new PersistentField<>(() -> durableBefore,
                                                           (input, prev) -> {
-                                                              if (input.fullyContainedIn(prev))
-                                                              {
-                                                                  Invariants.expect(prev.equals(DurableBefore.merge(input, prev)));
+                                                              DurableBefore next = DurableBefore.merge(input, prev);
```

Comment:
to be honest I was hoping for a check that is cheaper than full durable before comparison but looks like fully contained in did not work?